src/odemis/driver/technolution.py

```diff
@@ -907,8 +917,8 @@ def __init__(self, name, role, parent, **kwargs):
                 tuple(tuple(0 for i in range(0, self.shape[0])) for i in range(0, self.shape[1]))
                 , setter=self._setCellDarkOffset)  # Holds only ints
         self.cellDigitalGain = model.TupleVA(
-                tuple(tuple(1.2 for i in range(0, self.shape[0])) for i in range(0, self.shape[1]))
-                , setter=self._setCellDigitalGain)  # Holds floats and converts inputted ints to floats
+                tuple(tuple(1.2 for i in range(0, self.shape[0])) for i in range(0, self.shape[1])),
+                setter=self._setCellDigitalGain)  # Holds floats and converts inputted ints to floats
```

Comment:
hold -> contain
inputted -> input (inputted is not incorrect, but more commonly used is input as past tense)

Reply:
On second thought, I don't think you need this comment, as the default value is clearly a float.

Reply:
The comment was indeed not correct. It was there to say that the setter will convert int numbers which are used as input to floats. I changed it to:
>  # Holds floats, if input holds ints these are these are converted

Reply:
Read that sentence a second time ;)
Please skip the comment. The type should be explained in the respective setters. I just checked and it is not explained in the setters yet. The applies for the two VAs above.

Reply:
In this setters there is a comment and logging message but looking at it again those setters should have better dostrings and comments. I will need to look at that in review/branch of sim2.
The other two setters actually don't convert from ints to floats because these don't hold floats.
I skipped the comment.